docs/userguide.rst

```diff
@@ -116,6 +116,13 @@ There are several convenient environment variables you can set when working with
 
    $ export REANA_SERVER_URL=http://reana.cern.ch
 
+- ``REANA_ACCESS_TOKEN`` Identifies the current user when performing
+  protected actions.
+
+.. code-block:: console
+
+   $ export REANA_ACCESS_TOKEN=<ACCESS-TOKEN>
```

Comment:
Please use `REANA_ACCESS_TOKEN=XXXXXXX ` value and the explanatory verbiage from the REANA-Client.

(because <...> could give a syntax error for shell if people copy and paste things)